cmd/util.go

```diff
@@ -38,18 +37,20 @@ func readAndValidateConfig(fs afero.Fs, configFile string, verbose bool) (*confi
 	return config, err
 }
 
-func exitOnConfigErrors(err error) {
+func mergeConfigValidationErrors(err error) error {
 	if err != nil {
 		fmt.Println("fogg.json has error(s):")
-		errs, ok := err.(validator.ValidationErrors)
+		validatonErrors, ok := err.(validator.ValidationErrors)
 		if ok {
-			for _, err := range errs {
-				msg := fmt.Sprintf("\t%s is a %s %s", err.Namespace(), err.Tag(), err.Kind())
-				fmt.Println(strings.Replace(msg, "Config.", "", 1))
+			var sb strings.Builder
+			for _, err := range validatonErrors {
```

Comment:
a project I sometimes use for this is https://github.com/hashicorp/go-multierror